### ISSUE TITLE
Fozruk not dropping Rod of Order from Breaking they keystone quest

### ISSUE THREAD

**Issue by pauliusrug** [dalaranwow/dalaran-wow#5521]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I was killing fozruk, Killed him and did not ge the drop. I was killing it with a lvl 37 dwarf protector.

**Expected behaviour**: Tell us what should happen instead
I should have gotten the drop

**Steps to reproduce the problem**:

1. Kill Fozruk
2. Not get the 100% drop 
3. 

**Include proofs for this behaviour**
http://prntscr.com/jwst4s
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Works as intended. Most likely it was an addon issue or you were in a raid group.

Closing
